### DISCUSSION TITLE
Add challenging levelling modes

### DISCUSSION THREAD

**Discussion by trinitarius** [chromiecraft/chromiecraft#4144] [Suggestions]
**Motivation behind this suggestion:** Incentivise people to level more characters and attract new players to the server that like a challenge with exclusive rewards. There are so many challenging modes on other private servers that are used to enhance replayability (which in my opinion is very important regarding the philosophy of CC) and to give players a possibility to find appreciation in their achievements.

Realisable without additional patches: add an NPC that has either a dialogue or quest option with either items that turn on challenges or by receiving a permanent buff that marks people for any challenge mode.

**Examples** of such challenge modes could be (of which people could choose/combine freely and which could but don't have to end at max level (currently level 60)):
- Reduced XP gain (0.25 or 0.5 experience gain to slowly go through all of the quests/content of WoW)
- Forced PvP/War Mode (while having PvP always on, players could receive an experience boost for quests to incentivise playing with PvP in the open world, or higher honor gain for kills in world PvP)
- Classic Hardcore/Ironman (no RDF, permadeath (delete/lock dead characters), no grouping/trading apart from it's own community, no AH, people could become Immortals for max level and decide after each max level bracket if they want to continue as Hardcore)
- "Softcore" (an easier version of Hardcore in which players don't get their characters locked/deleted after death but have some other major loss (all items/gold lost, tp to starter area, experience loss etc.)
- Criminal/Outlaw (Horde and Alliance factions start at Hated, can only level by mob kills/dungeons/neutral quests and by help from other players, can farm reputation for main factions later with certain turn in quests, can decide for Alliance or Horde, could be lore-implemented with Defias Brotherhood, Scarlet Crusade etc. pledging allegiance to those with an NPC in respective starter area)
- "Selfmade/Professionist/Workingman" (can only use and equip items that have been self-crafted until level 60 or even beyond)
- Hard Progression (needs to finish quests/zones/dungeons/raids in a certain order first)
etc.

**Examples** of exclusive rewards for above challenges that are not immersion breaking:
- Titles (Resilient, Challenger, Hardcore, Mortal, Immortal, Ironman, Bandit, Defias, Scarlet Crusader, Selfmade etc.)
- Pets (depending on how many exclusive models exist in a normal WotLK client)
- Mounts (same as above)
- Tabards (same as above)
- Iconic Transmog (e.g. unobtainable items)
- Toys/Morphs/Fun items 
- ChromiePoints
- Leaderboards 
etc.

**Comment by balleny** [comment]
IMHO half of the ideas you can enforce yourself or within a "community". the other half sounds more like a fun/custom server and would require plenty additional modifications
-> not in favour.

**Reply by trinitarius** [reply]
I understand the criticism for it being too "custom". The whole idea is however that this is supposed to be enforced by the server rather than by a community simply for the reason that people are hesitant to interact with such communities except it is already supported by the server. It leads to more people trying these challenges that would otherwise not do it. Other servers have experienced such a development as well (in one instance a PvE server with dead world PvP got one of the most interacting world PvP communities after adding such a mode).

**Reply by balleny** [reply]
ye, but the point of CC is not forcing people into other types of play, but being close to blizzlike to fix the defaults

**Reply by trinitarius** [reply]
Nothing I wrote was about forcing anybody. "Enforcing" was meant as having the server to implement those modes to give more players an easier way to do those challenges as well as making other people aware. It will give people something to do apart from raid-logging which will make the server seem more alive to new players and helps them get groups and queues. Also it could revive certain factors, such as world PvP.

"Blizzlike" on the other hand is a term of which the meaning is still discussed. Having cross-faction, tuning of raids, balancing PvP rewards, gatekeeping certain content, skipping raids, adding vanilla content or reverting Blizzard's faults is what CC is all about and far away from a strict "blizzlike"-server. WSG's trees were initially not "los" back in WotLK and thus "blizzlike", after heavy backlash by the PvP community CC reverted this change though from the "blizzlike" version to a private server version that makes more sense since it really gives the BG items the intended purpose. 

Having optional levelling modes does not force anybody to do these and people can opt-in or -out of these. This is the same as with the transmog function. There was transmog before on CC as well and a small niche community but since it was implemented/enforced by the server as an optional feature way more people do it nowadays and it is actually a bigger topic/comunity now on CC.

**Comment by mrmacman65** [comment]
Do the addons/files affect your other toons?  Also, why do you ban the ah, but allow professions, elixirs, potions, enchantments, etc...?
